src/status_im/accounts/recover/core.cljs

```diff
@@ -35,8 +36,15 @@
      ;; don't want secure data to be printed
      (let [data (-> (types/json->clj result)
                     (dissoc :mnemonic)
-                    (types/clj->json))]
-       (re-frame/dispatch [:accounts.recover.callback/recover-account-success data password])))))
+                    (types/clj->json))
+           address ((types/json->clj result) :address)
```

Comment:
As there are many `(types/json->clj result)` calls, this intermediate result should be captured in a binding

Reply:
Done!